app/tests/spec/views/settings/display_name.js

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'jquery',
+  'sinon',
+  'lib/promise',
+  'views/settings/display_name',
+  'lib/metrics',
+  'models/notifications',
+  'models/reliers/relier',
+  'models/user',
+  '../../../mocks/router',
+  '../../../lib/helpers'
+],
+function (chai, $, sinon, p, View, Metrics, Notifications, Relier, User, RouterMock, TestHelpers) {
+  'use strict';
+
+  var assert = chai.assert;
+
+  describe('views/settings/display_name', function () {
+    var view;
+    var routerMock;
+    var metrics;
+    var user;
+    var email;
+    var account;
+    var relier;
+    var notifications;
+
+    beforeEach(function () {
+      email = TestHelpers.createEmail();
+      routerMock = new RouterMock();
+      metrics = new Metrics();
+      user = new User();
+      relier = new Relier();
+      notifications = new Notifications();
+      account = user.initAccount({
+        email: email,
+        uid: 'uid',
+        verified: true,
+        sessionToken: 'fake session token'
+      });
+    });
+
+    afterEach(function () {
+      metrics.destroy();
+
+      view.remove();
+      view.destroy();
+
+      view = metrics = null;
+    });
+
+    function initView () {
+      view = new View({
+        router: routerMock,
+        metrics: metrics,
+        user: user,
+        relier: relier,
+        notifications: notifications,
+        screenName: 'display-name'
+      });
+
+      sinon.stub(view, 'getSignedInAccount', function () {
+        return account;
+      });
+
+      sinon.stub(view, 'isUserAuthorized', function () {
+        return p(true);
+      });
+      sinon.stub(account, 'fetchProfile', function () {
+        return p();
+      });
+      sinon.stub(user, 'setAccount', function () {
+        return p();
+      });
+
+      return view.render()
+        .then(function () {
+          $('#container').html(view.el);
+        });
+    }
+
+    describe('renders', function () {
+      it('redirects to /settings/avatar/gravatar if permissions already granted', function () {
+        var name = 'joe cool';
+        account.set('displayName', name);
+
+        return initView()
+          .then(function () {
+            assert.isTrue(account.fetchProfile.called);
+            assert.isTrue(user.setAccount.calledWith(account));
+            assert.equal(view.$('.text').val(), name);
```

Comment:
You could always do a `view.getElementValue('.text')`. And, same comment as above [about the selector `.text`](https://github.com/mozilla/fxa-content-server/pull/2748/files#r36174263).